packages/spectacular/README.md

```diff
@@ -4,43 +4,65 @@
+
 ## Application testing
 
+Spectacular's application testing API configures the Angular testing module and
+bootstraps a component while ensuring that all application-level hooks are run.
+
+The application test harness is used to test configuration Angular modules,
+bootstrap listeners, and application initializers.
+
 ### Public API
 
 | Export name                       | Kind                                        | Description                                                                                                                                                       |
 | --------------------------------- | ------------------------------------------- | ----------------------------------------------------------------------------------------------------------------------------------------------------------------- |
 | `createApplicationHarness`        | Factory for `SpectacularApplicationHarness` | Bootstrap a Spectacular application with the specified metadata. Useful to test configuration Angular modules, bootstrap listeners, and application initializers. |
 | `CreateApplicationHarnessOptions` | Options for `createApplicationHarness`      | Application harness options.                                                                                                                                      |
-| `SpectacularAppComponent`         | Bootstrapped component                      | A root component for testing.                                                                                                                                     |
-| `SpectacularApplicationHarness`   | Interface                                   | Application harness data structure.                                                                                                                               |
-| `spectacularAppTag`               | String                                      | The tag name for `SpectacularAppComponent`s DOM element.                                                                                                          |
+| `SpectacularApplicationHarness`   | Interface                                   | A harness for testing application-level software artifacts.                                                                                                       |
 
 ### Feature testing
 
-The feature testing API helps configure the Angular testing module and sets up a
-test harness for an Angular feature module. It contains a few companion services
-that wrap Angular's built-in navigation services, but adjust to the Angular
-feature module under test.
+Spectacular's feature testing API configures the Angular testing module and sets
+up a test harness for an Angular feature module. It contains a few companion
+services that wrap Angular's built-in navigation services, but adjusted to the
+Angular feature module under test.
+
+The feature test harness is used to test feature modules and shell modules.
```

Comment:
This excerpt doesn't mention anything about testing the feature module routing. I think that's an important part of the feature testing if not the most important.

We should emphasize in that.

Reply:
That depends on the structure of the routing module. Does it import all its routed component dependencies?

A feature module on the other hand either has routing itself or imports a routing module. It's the entry point from a higher-level route.

Reply:
I see your point, nonetheless, I think we should make more clear what problem are we solving with these tools or what do we think these are more useful.

Maybe this section is not the place though.

Reply:
Let's change it to a *routed feature module*.

Reply:
That sounds better, after all the API expects a `feature-path`.

Reply:
Have another look 🙂